flink-runtime/src/test/java/org/apache/flink/runtime/checkpoint/channel/ChannelStateWriterImplTest.java

```diff
@@ -36,52 +35,49 @@
 
 import static org.apache.flink.runtime.state.ChannelPersistenceITCase.getStreamFactoryFactory;
 import static org.apache.flink.util.CloseableIterator.ofElements;
-import static org.apache.flink.util.ExceptionUtils.findThrowable;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** {@link ChannelStateWriterImpl} lifecycle tests. */
-public class ChannelStateWriterImplTest {
+class ChannelStateWriterImplTest {
```

Comment:
nit: I cannot comment on that line because it wasn't touched in this PR but we could also change the signature of `runWithSyncWorker(BiConsumerWithException<ChannelStateWriter, SyncChannelStateWriteRequestExecutor, Exception> testFn)` to `runWithSyncWorker(BiConsumerWithException<ChannelStateWriter, SyncChannelStateWriteRequestExecutor> testFn)` since the exception throwing is not utilized anymore when relying on a more specific `assertThatThrownBy` in each test.

Reply:
After tried, I found we cannot change it.

Sometimes, `testFn` calls the ` worker.processAllRequests()`, and ` worker.processAllRequests()` won't throw exception, so `testFn` cannot be changed from `BiConsumerWithException` to `BiConsumer`.

Reply:
ok, fair point. :+1: